cip/cip25/src/main/java/com/bloxbean/cardano/client/cip/cip25/NFTProperties.java

```diff
@@ -44,22 +44,30 @@ public String getProperty(String name) {
         return (String) get(name);
     }
 
-    public NFTProperties property(String name, java.util.Map<String, String> values) {
+    public NFTProperties property(String name, java.util.Map<String, Object> values) {
```

Comment:
Can you please add javadoc with info like kind of values expected? String and List etc.

Reply:
Also, if the value is not String or List, should we throw IllegalArgumentException instead of ignoring it?

Reply:
> Can you please add javadoc with info like kind of values expected? String and List etc.

Sure.
Values expected are only String or List<String>

> Also, if the value is not String or List, should we throw IllegalArgumentException instead of ignoring it?

Sure good idea!